tests/unit/Logger/Logger/ConstructCest.php

```diff
@@ -111,9 +111,9 @@ public function loggerConstructStreamWithJsonConstants(UnitTester $I)
 
         $I->seeInThisFile($expected);
 
-        $I->safeDeleteFile(
-            $outputPath . $fileName
-        );
+//        $I->safeDeleteFile(
```

Comment:
Can we clean this up? Or should we keep this?

Reply:
Yeah I will uncomment this